.docker/privnet-entrypoint.ps1

```diff
@@ -0,0 +1,15 @@
+#!C:\Program Files\PowerShell\7\pwsh.EXE -File
```

Comment:
@roman-khimov, I'm not sure about this shebang, because it works fine with our `Dockerfile.wsc` and windowsservercore-22 base image, but if someone desides to reuse it for another base image, then powershell executable location may differs from the specified one.

At the same time, it's not nice to go without this shebang.